man/ocitools-generate.1.md

```diff
@@ -137,18 +137,21 @@ read the configuration from `config.json`.
   The special *PATH* `host` removes any existing PID namespace from
   the configuration.
 
-**--poststart**=CMD
-  Path to command to run in poststart hooks. This command will be run before
-  the container process gets launched but after the container environment and
-  main process has been created.
-
-**--poststop**=CMD
-  Path to command to run in poststop hooks. The command will be run
-  after the container process is stopped.
-
-**--prestart**=CMD
-  Path to command to run in prestart hooks. This command will be run before
-  the container process gets launched but after the container environment.
+**--poststart**=CMD[:ARGS..]
+  Set command to run in poststart hooks. Can be specified multiple times.
+  The multiple commands will be run in order before the container process
+  gets launched but after the container environment and main process has been
+  created.
+
+**--poststop**=CMD[:ARGS...]
+  Set command to run in poststop hooks. Can be specified multiple times.
+  The multiple commands will be run in order after the container process
+  is stopped.
+
+**--prestart**=CMD[:ARGS...]
+  Set commands to run in prestart hooks. Can be specified multiple times.
+  The multiple commands will be run in order before the container process
+  gets launched but after the container environment.
```

Comment:
These are run after the container process has been created but before it executes the user-configured code (opencontainers/runtime-spec#466).  Although, again, with opencontainers/runtime-spec#483 presumably landing soon it may not be worth spending time polishing this up.